src/indexed_crate.rs

```diff
@@ -1260,6 +1260,31 @@ mod tests {
             }
         }
 
+        #[test]
+        fn trait_types() {
```

Comment:
Please keep the test case name matching the test crate name, so it's easier to correlate tests to the data they use.